src/main/java/com/mallowigi/icons/TintedIconsComponent.java

```diff
@@ -139,10 +137,20 @@ private static void refreshColors() {
 
     @NotNull
     @Override
-    public SVGLoader.SvgElementColorPatcher forURL(@Nullable final URL url) {
```

Comment:
I removed this deprecated method, because the current version only supports 2021.1+. So I just changed it to the new one. Figured I'd make the plugin verifier complain less, (even though now it's going tho complain more about internal API usages :sweat_smile: )